packages/ingest/recursion.js

```diff
@@ -18,7 +18,7 @@ const log = require('@cumulus/common/log');
 async function recursion(fn, originalPath, currentPath = null, position = 0) {
   // build the recursion path object
   const regex = /(\(.*?\))/g;
-  const rules = originalPath.split(regex).map((i) => i.replace(/\\\\/g, '\\'));
+  const rules = (originalPath || '/').split(regex).map((i) => i.replace(/\\\\/g, '\\'));
```

Comment:
I'd like some feedback on whether this is a good idea. This is the most centralized way I could think of to fix the ftp/sftp `list`s failing on an empty path, but I'm not convinced that it is the best way to do so. My understanding of `recursion.js` in combination with `[s]ftp.list()` does not feel 💯

Reply:
In what order do all the constructors get called in? I'm wondering if maybe we should do this in the constructor of `ftpMixin`. Maybe @yjpa7145 has some opinions on this one too.

Reply:
Having not touched `recursion.js`, this seems reasonable.  It's pretty disgusting of Javascript that you can't set `'/'` as the default for the `originalPath` parameter, but an empty string still evaluates to false.